app/routes/component-tree.js

```diff
@@ -0,0 +1,47 @@
+import TabRoute from "ember-inspector/routes/tab";
+
+export default TabRoute.extend({
+  model() {
+    return [];
+  },
+
+  setupController() {
+    this._super(...arguments);
+    this.get('port').on('view:viewTree', this, this.setViewTree);
+    this.get('port').on('view:stopInspecting', this, this.stopInspecting);
+    this.get('port').on('view:startInspecting', this, this.startInspecting);
+    this.get('port').on('view:inspectDOMElement', this, this.inspectDOMElement);
+    this.get('port').on('view:inspectComponent', this, this.inspectComponent);
+    this.get('port').send('view:setOptions', { options: this.get('controller.options') });
+    this.get('port').send('view:getTree');
+  },
+
+  deactivate() {
+    this.get('port').off('view:viewTree', this, this.setViewTree);
+    this.get('port').off('view:stopInspecting', this, this.stopInspecting);
+    this.get('port').off('view:startInspecting', this, this.startInspecting);
+    this.get('port').off('view:inspectDOMElement', this, this.inspectDOMElement);
+    this.get('port').off('view:inspectComponent', this, this.inspectComponent);
+
+  },
+
+  setViewTree(options) {
+    this.set('controller.viewTree', options.tree);
+  },
+
+  startInspecting() {
```

Comment:
Could we combine `startInspecting` and `stopInspecting` into one, perhaps `toggleInspecting` and pass a boolean for true or false?